glossary.yml

```
@@ -6057,6 +6057,11 @@
     def: >
       المُتغير هو موقع يٌخزِن البيانات المؤقتة داخل البرنامج ويمكن تعديله وتخزينه وعرضه
       عند الحاجة.
+  ja:
+    term: "変数 (プログラム)"
+    def: >
+      プログラム内にデータが結合するの名前です。変数の値は定義したの後で代再入出来のこと。
+      変数は英語で「variable」(ヴァリアブル)と呼んでいのこと。
```

Change to plain form as suggested.

```suggestion
      変数は英語で「variable」(ヴァリアブル)という。
```